src/Graphics/OpenGLContext/opengl_ContextImpl.cpp

```diff
@@ -116,9 +117,17 @@ void ContextImpl::enable(graphics::EnableParam _parameter, bool _enable)
 	m_cachedFunctions->getCachedEnable(_parameter)->enable(_enable);
 }
 
-u32 ContextImpl::isEnabled(graphics::EnableParam _parameter)
+void ContextImpl::polygonOffsetEnable(bool _enable)
 {
-	return m_cachedFunctions->getCachedEnable(_parameter)->get();
+	if (config.generalEmulation.enableFragmentDepthWrite == 0 && config.frameBufferEmulation.N64DepthCompare == 0)
```

Comment:
This condition looks redundant. 
if config.frameBufferEmulation.N64DepthCompare != 0, _enable is always false, see GraphicsDrawer.cpp line 1665
Thus, if (config.generalEmulation.enableFragmentDepthWrite == 0) should be enough.